Screens/Profile.tsx

```diff
@@ -0,0 +1,293 @@
+                                        />
+                                        <TouchableOpacity
+                                            onPress={() =>
+                                                console.log(
+                                                    "change profile picture"
+                                                )
+                                            }
+                                            style={styles.addImageButton}
+                                        >
+                                            <MaterialCommunityIcons
+                                                name="camera-plus-outline"
+                                                style={styles.icon}
+                                            />
+                                        </TouchableOpacity>
+                                    </ImageBackground>
+                                </View>
+
+                                <Text style={styles.label}> Name</Text>
+                                <TextInput
+                                    style={styles.input}
+                                    onBlur={handleBlur("name")}
+                                    onChangeText={handleChange("name")}
+                                    autoCapitalize={"words"}
+                                    value={values.name}
+                                />
+
+                                {errors.name ? (
+                                    <Text style={styles.errors}>
+                                        {errors.name}
+                                    </Text>
+                                ) : null}
+
+                                <Text style={styles.label}> Email</Text>
+
+                                <TextInput
+                                    style={styles.input}
+                                    onBlur={handleBlur("email")}
+                                    onChangeText={handleChange("email")}
+                                    keyboardType={"email-address"}
+                                    value={values.email}
+                                />
+
+                                {errors.email ? (
+                                    <Text style={styles.errors}>
+                                        {errors.email}
+                                    </Text>
+                                ) : null}
+
+                                <Text style={styles.label}> Phone</Text>
+
+                                <TextInput
+                                    style={styles.input}
+                                    onBlur={handleBlur("phone")}
+                                    keyboardType={"phone-pad"}
+                                    onChangeText={handleChange("phone")}
+                                    value={values.phone}
+                                />
+
+                                {errors.phone ? (
```

Comment:
all of these errors displays are the same shape, I wonder if there is a way to create a component that would take care of these and save some of the repetition?